src/IssueViz.Security/IssueVizSecurityPackage.cs

```diff
@@ -36,9 +34,17 @@
 
 namespace SonarLint.VisualStudio.IssueVisualization.Security
 {
+    /*
+        This is a simple package that just provides the basic VS plumbing for the security commands
+        and tool windows.
+
+        It doesn't provide any other services, and doesn't need to be auto-loadable. It will be loaded
+        by VS only when needed i.e. when it has received a reqeust to to invoke a command or display a
```

Comment:
reqeust  = typo.

But I don't think this comment is needed, we have other packages without AutoLoad (i.e. IssueVizPackage) so it's not uncommon.

Reply:
I don't think we have a clear enough idea what specific packages are for or what there load characteristics are expected to be. I'm not even sure how many packages we have any more. However, my feeling is that we have too many, we don't understand what the relationships between them are, and some of them will be loading unnecessarily/sooner than necessary. This isn't something we can address in the short term. However, I'm going to start adding comments to packages as part of an effort to understand what we currently have.